examples/mobile/UIComponents/components/dialogs/dialog.html

```diff
@@ -93,13 +93,10 @@ <h1>
 					consectetur adiptiscing elit, sed
 					doei usmod tempore sit ipsum
 				</div>
-				<div class="ui-popup-footer ui-grid-col-2">
-					<a class="ui-btn" data-rel="back">
-						Action1
-					</a>
-					<a class="ui-btn" data-rel="back">
-						Action2
-					</a>
+				<div class="ui-popup-footer">
+					<button class="ui-btn" data-style="flat">Action 1</button>
```

Comment:
How about to remain the date-rel="back" ?
date-rel="back" means , if push the button , the popup is closed?

Reply:
date-rel = "back" works on tag a. It doesn't work in button tag.